meta-iot2000-example/recipes-internal/imageversionfile/files/image-release

```diff
@@ -1 +1 @@
-IMAGE_RELEASE="V2.1.0"
+IMAGE_RELEASE="V2.1.2"
```

Comment:
Has some build of this revision already been distributed under "v2.1.2"? Or why does this tag not include the succeeding commits?